acceptance/new-user/join-button.test.js

```diff
@@ -0,0 +1,66 @@
+/** @jest-environment jsdom */
+const puppeteer = require("puppeteer");
+const URLS = require("../../constants/urls");
+const { JOIN } = URLS;
+const { delay } = require("../../utils/delay");
+
+let browser, page;
+
+function confirmAlerts() {
+  page.on("dialog", async (dialog) => {
+    await dialog.accept();
+
+    delay(2000);
```

Comment:
What is the use of delay here?

Reply:
It allows proper navigation. It is necessary I tried without delay , but it cause navigation issue. OR evaluate assertion too early.

Reply:
What will happen if the navigation sometimes takes more than 2s
And we will always have to wait for a 2s delay even if the navigation happens within 200ms?